clang/lib/CConv/ConstraintBuilder.cpp

```diff
@@ -202,6 +210,11 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
             } else
               ArgumentConstraints = CB.getExprConstraintVars(A);
 
+            if (callUntyped) {
+              deferred.push_back(ArgumentConstraints);
+              continue;
+            }
```

Comment:
I think it's clearer if you drop the `continue` here, make the next `if` an `else if`. (Yes, this goes back on my earlier suggestion!)

Reply:
Will do